policies/clair/vulnerabilities-check.rego

```diff
@@ -1,42 +1,45 @@
 package required_checks
 
-violation_critical_vulnerabilities[{"msg": msg, "details":{"name": name, "description": description, "url": url}}] {
+violation_critical_vulnerabilities[{"msg": msg, "vulnerabilities-count": vulns_count, "details":{"name": name, "description": description, "url": url}}] {
   rpms_with_critical_vulnerabilities := {rpm.Name | rpm := input.data[_].Features[_]; count(rpm.Vulnerabilities) > 0; rpm.Vulnerabilities[_].Severity == "Critical"}
   not count(rpms_with_critical_vulnerabilities) == 0
 
   name := "clair_critical_vulnerabilities"
-  msg = sprintf("Found packages with critical vulnerabilities: %s", [concat(", ", rpms_with_critical_vulnerabilities)])
-  description := "The image musn't contain packages that have critical severity vulnerabilities."
-  url := "https://source.redhat.com/groups/public/product-security/content/product_security_folder/understanding_red_hat_security_vulnerabilities_10pdf"
+  vulns_count = count(rpms_with_critical_vulnerabilities)
+  msg = sprintf("Found packages with critical vulnerabilities. Consider updating to newer version of those packages, they may not longet be affected by the reported CVEs. You can check clair logs for more infromation. Packages found: %s", [concat(", ", rpms_with_critical_vulnerabilities)])
+  description := "The image shouldn't contain packages that have critical severity vulnerabilities."
```

Comment:
How will msg and description work together? Would it make sense to split the information where msg states that a category check found affected packages, with the recommendation to update them, and the packages are listed as the description? 

"The image shouldn't contain packages that have <category> severity vulnerabilities" doesn't seem that useful to me if we have the new msg text.

Reply:
Sounds about right, MSG in current state would be too overwhelming.
Agreed on: MSG - states a category check found packages with recommendation
DESCRIPTION: list all the packages
If you agree I would replace the whole sentence in description with affected packages.